docs/specs/src/hwp_3_correctness.md

```diff
@@ -0,0 +1,404 @@
+
+System Correctness: Confidence
+==============================
+
+In the frame of the Byzantine Generals problem, the correctness of a
+distributed coordination system is analyzed through the lens of "fault
+tolerance". In our frame we take on a broader scope and address the
+question of the many kinds of confidence necessary for a system adoption
+and continued use. We identify and address the following dimensions
+necessary confidence:
+
+1.  **Fault Tolerance:** the system's resilience to external
+  perturbations, both malicious and natural. Intrinsic integrity.
+
+2.  **Completeness/Fit:** the system's apriori design elements that
+  demonstrate fitness for purpose. We demonstrate this by describing
+  how Holochain addresses, multi-agent reality binding, scalability,
+  and shared-state finality.
+
+3.  **Security:** the system's resilience to intentional disruption by
+  malicious action.
```

Comment:
point 1, Fault Tolerance, already covers malicious disruption, so how is this different?

Reply:
It feels to me like there's a qualitative difference between gaming behavior and ability to gate access to parts of the systems through things like our capabilities model, that is of a different order of intentional distruption than the kind of fault-tolerance that happens from cosmic rays that flips bits (which you could also do intentionally).  The only reason I put "intentional" under fault tolerance is that one can't necessarily know if a bit-flip disruption event was intentional or random, but one can pretty easily know that a cosimic ray didn't orchestrate a 51% attack.

This is meant to make the distinction between those two types.  Any idea how better to describe that?